lib/ramble/docs/tutorials.rst

```diff
@@ -19,5 +19,5 @@ various features.
     :maxdepth: 1
     :caption: Tutorials
 
-    tutorials/general_usage
+    tutorials/running_an_experiment
```

Comment:
Maybe the name should have gromacs in it, just in case we want to add other apps later.

like:
`running_a_simple_gromacs_experiment`